source/_base.css

```diff
@@ -11,6 +11,10 @@ Copyright (c) 2015 Daniel Eden
   animation-fill-mode: both;
 }
 
+.animated.centerAnimation{
```

Comment:
This doesn't appear to be present in the compiled animate.css, and it's also relying on a lot of other CSS properties to do what (I think) it's intended to do. Remove this whole declaration.